src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1311,11 +1311,7 @@
   <data name="OperatorOverloadsHaveNamedAlternatesCodeFixTitle" xml:space="preserve">
     <value>Add operator overload named alternate</value>
   </data>
-  <data name="AvoidMultipleEnumerationsMessage" xml:space="preserve">
-    <value>Possible multiple enumerations of 'IEnumerable' collection in this method. To prevent unnecessary enumerations, consider caching it first by calling a method such as 'ToList' or 'ToArray'.</value>
-    <comment>{Locked="IEnumerable"} {Locked="ToList"} {Locked="ToArray"}</comment>
-  </data>
-  <data name="AvoidMultipleEnumerationsTitle" xml:space="preserve">
+  <data name="PosssibleMultipleEnumerationsOfIEnumrableCollection" xml:space="preserve">
```

Comment:
I dont' think we should use same message for title and message, in general title should be shorter and descriptive as much as possible. Message could give more info than the title, how about:
`Avoid multiple enumerations` for title and ` Possible multiple enumerations of 'IEnumerable' collection in this method. Consider a design that avoid multiple enumerations`

Reply:
With the final period to avoid having the resource analyzer complaining (no period for single sentence, final period for multiple sentences) :P

Reply:
I will change it to 
`Possible multiple enumerations of 'IEnumerable' collection in this method. Consider using an implementation that avoid multiple enumerations`
(I add a word 'using')